Contrib/NP_Score/npscorer.py

```diff
@@ -13,11 +13,13 @@
 # peter ertl, august 2015
 #
 
+from __future__ import print_function
 from rdkit import Chem
 from rdkit.Chem import rdMolDescriptors
 import sys,math,gzip,pickle
+import os.path as op
```

Comment:
That is no problem, I just do that out of habit. Should I retract the PR and submit a new one?